node/resolvers/catalog/offer.ts

```diff
@@ -35,5 +35,12 @@ export const resolvers = {
     teasers: propOr([], 'Teasers'),
     giftSkuIds: propOr([], 'GiftSkuIds'),
     discountHighlights: propOr([], 'DiscountHighLight'),
+    spotPrice: (offer: Seller["commertialOffer"]) => {
+      const sellingPrice = offer.Price
+      const spotPrice: number | undefined = offer.Installments.find(({NumberOfInstallments, Value}) => {
```

Comment:
```suggestion
      const spotPrice: number | undefined = offer.Installments.find(({ NumberOfInstallments, Value }) => {
```

just to calm my lint anxiety

Reply:
Done my friend

It's all good now